ci/jenkinsfile.groovy

```diff
@@ -17,14 +17,32 @@
 
 import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 
-run_cpu = "bash ci/bash.sh mlcaidev/ci-cpu:a7b7a72"
-run_cuda = "bash ci/bash.sh mlcaidev/ci-cu121:a7b7a72"
-pkg_cuda = "bash ci/bash.sh mlcaidev/package-cu121:1da76a2"
+run_cpu = 'bash ci/bash.sh mlcaidev/ci-cpu:a7b7a72'
+run_cuda = 'bash ci/bash.sh mlcaidev/ci-cu121:a7b7a72'
+pkg_cuda = 'bash ci/bash.sh mlcaidev/package-cu121:1da76a2'
+
+cuda_target = 'cuda --arch=sm_86 --max_threads_per_block=1024 --max_num_threads=1024 --max_shared_memory_per_block=49152'
```

Comment:
a target should not be exposed in a CI environment, and instead, let's move them to the specific bash script